src/main/java/org/jenkinsci/test/acceptance/plugins/gradle/GradleInstallation.java

```diff
@@ -33,7 +34,19 @@ public GradleInstallation(Jenkins context, String path) {
         super(context, path);
     }
 
+    public static final String LATEST_VERSION = "latestVersion";
+
     public static void installGradle(Jenkins jenkins, String name, String version) {
```

Comment:
Please add java doc that that it is clear to others on how to use the version parameter (LATEST_VERSION).